### PR TITLE
Tool template

### PR DESCRIPTION
**Pull request by talyguryn**
Tool's files

### CODE REVIEW

File: template/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "main": "dist/bundle.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "webpack --mode production",
+    "build:dev": "webpack --mode development --watch",
+    "example": "open ./example/index.html"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.19.3",
+    "@babel/preset-env": "^7.19.4",
```
Comment (neSpecc) [inline template/package.json:14]: let's remove Babel, since TypeScript is replacing it

File: template/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    // "incremental": true,                         /* Enable incremental compilation */
+    "target": "es5",                                /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```
Comment (neSpecc) [inline template/tsconfig.json:7]: that is why we don't need Babel

File: template/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    "module": "commonjs",                           /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
+    "lib": ["ES2020", "dom"],                       /* Specify library files to be included in the compilation. */
+    // "allowJs": true,                             /* Allow javascript files to be compiled. */
```
Comment (neSpecc) [inline template/tsconfig.json:10]: maybe it would be better to remove unused props?

File: template/webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+module.exports = {
+  /** Define entry point */
+  entry: path.resolve(__dirname, 'src', 'index.ts'),
+
+  /** Define bundle params */
+  output: {
+    // Define output path and filename
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'bundle.js',
```
Comment (neSpecc) [inline template/webpack.config.js:11]: lets use tool name here

File: template/tsconfig.json (hunk 3 above)
Comment (talyguryn) [reply template/tsconfig.json:10]: i decided to left them as usual to allow user configure this file as he needs

**Pull request merged**